manual_tests/behaviors/lock_on_pr.py

```diff
@@ -5,10 +5,5 @@
 
 
 def test(pr: tacos_demo.PR) -> None:
-    for slice in pr.slices:
-        assert (
-            pr.check("Terraform Lock", f"tacos-gha / main ({slice})")
-            .wait()
-            .success
-        )
+    assert pr.check("tacos_lock").wait().success
```

Comment:
Did something change that I am not seeing? Why are we no longer checking locking per slice?

Reply:
changes in slice.py get us much closer to the old semantics: `\.check("X")` will match any job where `run.job.startswith("X")`, with `run.job` defined as the first bit after `" / "` in the check name. In total, it matches all of the tacos_lock jobs at once.

`.wait` is modified to return just one of the jobs matched, as long as they've all finished, with priority given to failed runs.

In total, it means that this line asserts that _all_ "tacos_lock" checks have finished and succeeded.